DOCS.md

```diff
@@ -54,19 +54,27 @@ pipeline:
 +   sensitive: true
 ```
 
-Example configuration with state tracked via remote:
+Example configuration with state tracked via remote.  You will need a file
```

Comment:
I linked to how backend configuration looks in terraform via their docs.  Then showed how to pass additional configs if necessary via `.drone.yml`.  Thanks for the note, I think this makes things more clear.